CHANGELOG.md

```diff
@@ -4,6 +4,112 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.36.0] - 2024-05-16
+
+This release adds a few new features, which I'll go over below in more detail.
+
+### [`subxt-core`](https://github.com/paritytech/subxt/pull/1508)
+
+We now have a brand new `subxt-core` crate, which is `#[no-std]` compatible, and contains a lot of the core logic that is needed in Subxt. Using this crate, you can do things in a no-std environment like:
+
+- `blocks`: decode and explore block bodies.
+- `constants`: access and validate the constant addresses in some metadata.
+- `custom_values`: access and validate the custom value addresses in some metadata.
+- `metadata`: decode bytes into the metadata used throughout this library.
+- `storage`: construct storage request payloads and decode the results you'd get back.
+- `tx`: construct and sign transactions (extrinsics).
+- `runtime_api`: construct runtime API request payloads and decode the results you'd get back.
+- `events`: decode and explore events.
+
+Check out [the docs](https://docs.rs/subxt-core/latest/subxt_core/) for more, including examples of each case.
+
+A breaking change that comes from migrating a bunch of logic to this new crate is that the `ExtrinsicParams` trait is now handed `&ClientState<T>` rather than a `Client`. `ClientState` is just a concrete struct containing the state that one needs for things like signed extensions.
+
+### [Support for reconnecting](https://github.com/paritytech/subxt/pull/1505)
+
+We've baked in a bunch of support for automatically reconnecting after a connection loss into Subxt. This comes in three parts:
+1. An RPC client that is capable of reconnecting. This is gated behind the `unstable-reconnecting-rpc-client` feature flag at the moment, and
+2. Handling in the subxt Backends such that when the RPC client notifies it that it is reconnecting, the backend will transparently handle this behind the scenes, or else pass on a `DisconnectedWillReconnect` error to the user where it cannot.
```

Comment:
I would just like to another point or modify this one to make it clear that  `LegacyRpcMethods and UnstableRpcMethods` doesn't utilize the retry logic and users that are using those APIs who wants retry logic can wrap that call in `retry` or `retry_stream` helpers or provide their own custom stuff.

Reply:
Makes sense; lemme tweak it to be clearer :)

Reply:
Tweaked it; hopefully it's clearer now?

Reply:
Yupp, thanks